dsl/src/commonTest/kotlin/com/freeletics/flowredux/dsl/CollectStateWhileInStateTest.kt

```diff
@@ -0,0 +1,84 @@
+package com.freeletics.flowredux.dsl
+
+import app.cash.turbine.test
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.flow
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.time.ExperimentalTime
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.FlowPreview
+import kotlinx.coroutines.flow.flatMapConcat
+
+@OptIn(ExperimentalCoroutinesApi::class, FlowPreview::class, ExperimentalTime::class)
+class CollectStateWhileInStateTest {
```

Comment:
Nitpicking: I'm wondering if we should merge this tests with the other  `CollectWhileInStateTest` to only have one "collectWhileInState" file with all tests of all variations...

OR

if we should have better naming conventions (i.e. class under test with "Test" suffix, i.e. then `CollectInStateSideEffectBuilderTest` ... 

I lean more towards the first option (merge all `inState<S> { collectWhileInState(...) }` into 1 file ...

Reply:
I'm fine with merging 👍

Reply:
a3b7e45